setup.py

```diff
@@ -350,9 +338,10 @@ def prepare_cew_for_windows():
     license=PKG_LICENSE,
     keywords=PKG_KEYWORDS,
     classifiers=PKG_CLASSIFIERS,
+    cmdclass={'build_ext': build_ext},
+    setup_requires=['numpy>=1.9'],
```

Comment:
Create a ``PKG_SETUP_REQUIRES`` in ``setupmeta.py``, import it on top of ``setup.py``, and use it here.